android_kernel_tools.rc

```diff
@@ -149,15 +154,15 @@ function fboota()
     # Waiting to get adb
     echo '';
     echo '';
-    echo -e ' \e[1;37m[ Waiting for adb root... ]\e[0m';
+    echo -e '  fboota: \e[1;37m[ Waiting for adb root... ]\e[0m';
```

Comment:
fboota info could keep share the color and be Inside the [ -> "[ fboota: Waiting..."